docs/en/aws-firehose/aws-firehose-setup.asciidoc

```diff
@@ -103,8 +102,16 @@ A duration of 60-300s should be suitable for most use cases.
 
 * Elastic recommends setting the *`es_datastream_name`* parameter to help route data to the correct integration data streams.
 If this parameter is not specified, data is sent to the *`logs-generic-default`* data stream by default.
+
+[WARNING]
+====
+The default data stream will change to `logs-awsfirehose-default` in January 2024. *To avoid breaking changes, do not leave `es_datastream_name` empty.*
+If you want to try the new https://docs.elastic.co/integrations/awsfirehose[routing functionality], you should explicitly set `es_datastream_name` to `logs-awsfirehose-default`.
+====
+
 * You can use the *`es_datastream_name`* parameter to route documents to any data stream.
-When using Elastic integrations, you must set this parameter.
+When Amazon Kinesis Data Firehose integration is installed, routing will be done automatically with `es_datastream_name` sets to `logs-awsfirehose-default`.
```

Comment:
```suggestion
+
When using the https://docs.elastic.co/integrations/awsfirehose[Amazon Kinesis Data Firehose integration], set `es_datastream_name` to `logs-awsfirehose-default` to automatically route your data.
```

Reply:
These two `when`s are actually both under the same bullet point :)